packages/components/src/Avatar.tsx

```diff
@@ -0,0 +1,105 @@
+import type { IconProps } from '@launchpad-ui/icons';
+import type { VariantProps } from 'class-variance-authority';
+import type { ImgHTMLAttributes, RefObject, SVGAttributes } from 'react';
+
+import { Icon } from '@launchpad-ui/icons';
+import { cva, cx } from 'class-variance-authority';
+
+import styles from './styles/Avatar.module.css';
+import { useImageLoadingStatus } from './utils';
+
+const avatar = cva(styles.base, {
+	variants: {
+		size: {
+			small: styles.small,
+			medium: styles.medium,
+			large: styles.large,
+		},
+	},
+	defaultVariants: {
+		size: 'medium',
+	},
+});
+
+const colors = cva(null, {
+	variants: {
+		color: {
+			0: styles.yellow,
+			1: styles.blue,
+			2: styles.pink,
+			3: styles.cyan,
+			4: styles.purple,
+		},
+	},
+});
+
+interface AvatarVariants extends VariantProps<typeof avatar> {}
+
+interface AvatarProps extends ImgHTMLAttributes<HTMLImageElement>, AvatarVariants {
+	ref?: RefObject<HTMLImageElement | null>;
+}
+
+interface IconAvatarProps extends Omit<IconProps, 'size'>, AvatarVariants {
+	ref?: RefObject<SVGSVGElement | null>;
+}
+
+interface InitialsAvatarProps extends SVGAttributes<SVGElement>, AvatarVariants {}
+
+const Avatar = ({ className, children, size = 'medium', ref, src, ...props }: AvatarProps) => {
+	const status = useImageLoadingStatus(src);
+
+	if (status === 'error') {
+		return children ? (
+			<InitialsAvatar size={size}>{children}</InitialsAvatar>
+		) : (
+			<IconAvatar size={size} />
+		);
+	}
+
+	// biome-ignore lint/a11y/useAltText: <explanation>
```

Comment:
Will biome detect a violation if `Avatar` was not provided an `alt` tag? I assume its possible to configure it to do that (its possible with eslint).

Reply:
Yeah ignored since React types has it as optional `alt?: string | undefined`